### DISCUSSION TITLE
change to csv_files_to_load dictionary

### DISCUSSION THREAD

**Discussion by Samira-El** [jmriego/pipelinewise-target-bigquery#32] [General]
Hey @jmriego, I made a [PR](https://github.com/transferwise/pipelinewise-target-bigquery/pull/30) earlier to revamp CI and move away from CircleCI since we no longer want to use CCI at Wise, and while doing it I had to make pylint happy by making few changes. One of these changes is this: https://github.com/transferwise/pipelinewise-target-bigquery/pull/30/files#diff-ff318a6d5029dba063770c4d125983107cb7c55c06190a9173cbc9e24a5df202L204

I don't have a bigquery instance to test if this is a breaking change or not, but I also noticed that this dictionary `csv_files_to_load` is not being used anywhere, it's declared then a temp file gets created for every stream and that's it.

Do you mind looking into it?

**Comment by jmriego** [comment]
thanks a lot @Samira-El !
I'll have a look this week and let you know about it. When I started building the code I tried creating a csv file and it worked but was not fast enough so I ended up changing everything to avro. I'm pretty sure this is just something I missed to remove but I'll do proper testing.

Thanks for these changes! They will definitely make things easier to work with. Also, BigQuery has a free tier that should be enough for testing this tap. I'm happy to help you setting that up if that would be useful to you
